simulation_parameters/Constants.cs

```diff
@@ -1890,6 +1894,16 @@ public static class Constants
         [CELLULASE_ENZYME] = 4.5f,
     };
 
+    public static readonly HashSet<BiomeType> SurfaceBiomes = new()
```

Comment:
Instead of hardcoding surface patches should be detected from their depth values.